modules/ROOT/pages/vm/vm-reference.adoc

```diff
@@ -224,26 +223,26 @@ The flow in which the content is published cannot contain a Listener (`<vm:liste
 `<vm:listener>`
 
 
-A source that creates and listens on VM queues. VM queues are created by placing listeners on them, which is why this listener contains parameters for a queue's behavior, such as it being persistent or not, the max capacity, etc.
+A source that creates and listens on VM queues. VM queues are created by placing listeners on them, which is why this listener contains parameters for a queue's behavior, such as whether or not it is persistent, the max capacity, and so on.
 
 
 ==== Parameters
 [cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | *x*{nbsp}
-| Number Of Consumers a| Number |  The number of concurrent consumers to be placed on the queue. As the number of consumers increases, so does the speed at which this source pushes messages into the owning flow. The default is 4 concurrent consumers. |  4 | {nbsp}
+| Configuration | String | The name of the configuration to use. | | *x*
+| Number Of Consumers a| Number |  The number of concurrent consumers to place on the queue. As the number of consumers increases, so does the speed at which this source pushes messages into the owning flow. The default is 4 concurrent consumers. |  4 | 
 | Transactional Action a| Enumeration, one of:
 
 ** ALWAYS_BEGIN
-** NONE |  The type of beginning action that sources can take regarding transactions. |  NONE | {nbsp}
+** NONE |  The type of beginning action that sources can take regarding transactions. |  NONE | 
 | Transaction Type a| Enumeration, one of:
 
 ** LOCAL
-** XA |  The type of transaction to create. Availability will depend on the runtime version. |  LOCAL | {nbsp}
-| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  | {nbsp}
-| Queue Name a| String |  The name of the queue. |  | *x*{nbsp}
-| Timeout a| Number |  How long to wait on the queue to complete and operation (either publishing or consuming) before failing with a timeout error. The default is 5 seconds. |  5 | {nbsp}
+** XA |  The type of transaction to create. Availability will depend on the Mule version. |  LOCAL | 
+| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  | 
```

Comment:
Wording is redundant. I suggest this: "Defines a policy for processing the redelivery of a message."